tests/unit/serialization/compact_test.py

```diff
@@ -200,3 +209,41 @@ def test_schema_writer(self):
 
         for name, kind in fields:
             self.assertEqual(kind, schema.fields.get(name).kind)
+
+
+class NestedSerializerTest(unittest.TestCase):
+    def _register_schema(self, service, obj):
+        writer = SchemaWriter(type(obj).__name__)
+        fields = []
+        for kind in FIELD_KINDS:
+            name = kind.name.lower()
+            fun = getattr(writer, f"write_{kind.name.lower()}", None)
+            if fun is None:
+                continue
+            fun(name, None)
+            fields.append((name, kind))
+        schema = writer.build()
+        service.compact_stream_serializer.register_schema_to_type(schema, type(obj))
+
+    def _serialize(self, serializationService, obj):
```

Comment:
```suggestion
    def _serialize(self, serialization_service, obj):
```
we should use camel case

Reply:
Refactored at https://github.com/hazelcast/hazelcast-python-client/pull/574/commits/7dc27039f9764d3aa88134a3885745041d84b096.